posts/2022/02/making-gpus-accessible-to-pydata-ecosystem-via-array-api.md

```diff
@@ -0,0 +1,481 @@
+<!--
+.. title: Making GPUs accessible to PyData Ecosystem via Array API
```

Comment:
This title feels just slightly off. If I'd keep the message unchanged, it'd be:
_Making GPUs accessible to the the PyData Ecosystem via the Array API Standard_.

Which is very long. An alternative would be _Making GPUs accessible to the PyData Ecosystem_. Which is maybe a bit too generic. So I'd lean towards the first one. Thoughts?

Reply:
The first one sounds reasonable to me.